tbmalt/physics/dftb/feeds.py

```diff
@@ -168,6 +169,126 @@ def _off_site_blocks(self, atomic_idx_1: Array, atomic_idx_2: Array,
 
         return blks
 
+    def _pe_blocks(self, atomic_idx_1: Array, atomic_idx_2: Array,
```

Comment:
I think that we would want to consolidate this with other parts of the code. But that is a task for the clean up operation later on.

Reply:
Yeah, sure. Maybe we can find a way to simplify the codes later on.